mobile_app_connector/models/app_hub.py

```diff
@@ -71,12 +77,36 @@ def mobile_get_message(self, partner_id, **pagination):
             'product.product': products,
             'correspondence': letters,
         }
+        unpaid_data = {
+            'recurring.contract': unpaid
+        }
         # TODO handle pagination properly
         limit = int(pagination.get('limit', 1000))
         messages = available_tiles[:limit].render_tile(tile_data)
+        msg_tmp = self.env['mobile.app.tile'].search([
```

Comment:
Better to directly grab the tile instead of searching by name which may be changed in the interface:
```
msg_tmp = self.env.ref('mobile_app_connector.tile_gi7').render_tile(unpaid_data)
```

Reply:
Perfect, this is changed.